app/utilities/schema.py

```diff
@@ -180,4 +186,10 @@ def load_schema_from_url(survey_url, language_code):
 
 
 def get_schema_file_path(schema_name, language_code):
-    return SCHEMA_PATH_MAP.get(language_code, {}).get(schema_name)
+    return get_schema_path_map().get(language_code, {}).get(schema_name)
+
+
+def cache_questionnaire_schemas():
+    for language_code, schemas in get_schema_path_map(dirs=(SCHEMA_DIR,)).items():
```

Comment:
I think this would be simpler if instead of passing the directories we passed a boolean, something like `include_test_schemas`. It could be defaulted to `False` (production default), and passed `True` by the `schemas` endpoint call to get the test schemas included.

Reply:
Done 👍

Reply:
Nice, looks much better 🙂